Source: Anders Ossowicki
Upstream: no
Reason: external.t looks for gzip in the wrong place
diff --git a/t/external.t b/t/external.t
index 1d90d7f..59c19e7 100644
--- a/t/external.t
+++ b/t/external.t
@@ -4,7 +4,7 @@
 # This limits the testing to UNIX-like
 # systems but that should be enough.
 
-my $gzip = "/usr/bin/gzip";
+my $gzip = "/bin/gzip";
 
 unless( -x $gzip &&
         ":$ENV{PATH}:" =~ m!:/usr/bin:! &&
